packages/core/src/navigation/navigation-item.element.scss

```diff
@@ -9,20 +9,32 @@
 :host {
   --color: inherit;
   --line-height: inherit;
+  --focus-outline: Highlight solid var(--focus-width);
```

Comment:
Per conversation with accessibility and design - this was the most practical way to go with this change. Using the browser focus-ring means this can only use roving tabindex instead of aria-activedescendent. aria-activedescendent is used b/c there roving tabindex doen't play nice with the keyboard handlers.

Reply:
All components have a normalized focused selector available that applies the browser focus ring without requiring tabindex. You can add the `_has-focus` selector here and then you will get the native focus style without tabindex.

https://github.com/vmware/clarity/blob/next/packages/core/src/internal/base/base.element.scss#L60  

```scss
:host([tabindex='0']:focus),
:host([_focused]) .input,
:host([_has-focus]) {
  outline: Highlight solid $cds-global-space-2;
  outline-offset: $cds-global-space-1;
}

@media (-webkit-min-device-pixel-ratio: 0) {
  :host([tabindex='0']:focus),
  :host([_focused]) .input,
  :host([_has-focus]) {
    outline-color: -webkit-focus-ring-color;
  }
}
```

Reply:
I removed this line, it is dead code. I looked back into why this css is the way it is. Here is whats driving it:

1. There is an issue with using outline; namely because part of the root layout has overflow on it means that its really hard to get the starts and items outline consistent. Items inside the area with overflow cut off the outline on the x-axis. This results with a start or item having a top/bottom outline but no left/right outline. 
2. The `Highlight` color with light theme and the background color fails accessibility with the `--cds-alias-object-interaction-background-hover` color

After discussions with Colin and Santina I changed to 

- use the border so that nothing gets cut off due to overflow
- use the `--cds-alias-object-interaction-background-highlight`
- keep the `--cds-alias-object-interaction-background-hover`

The later two tokens pairing gives an accessible color contrast
![Screen Shot 2021-08-17 at 3 32 01 PM](https://user-images.githubusercontent.com/433692/129809583-718dd786-f62b-44e6-816f-e5d7df81f35d.png)

vs `Highlight + `--cds-alias-object-interaction-background-hover`
![Screen Shot 2021-08-17 at 3 31 02 PM](https://user-images.githubusercontent.com/433692/129809643-638027fd-5d0f-45bf-a8f2-985b77d37cba.png)

I'll push the removal change now.

Reply:
One other thing that was weird is that I do apply `_has-focus` to the elements via `@state` property with reflection. But, I don't get the outline applied to the elements.

Reply:
Not blocking the PR if a11y and design has given the thumbs up on this approach but wanted to add some things to note,

- You might be able to fix the outline being cut off using `outline-offset: -2px`. This pulls the outline into the element rather than outside. This is what a handful of our components do. 

- The `Highlight` outline color is not a particular color. It changes from browser to browser and will also change based on the device preferences such as prefers dark theme, high contrast mode, mobile touch and custom color styles. Highlight also will automatically adjust its contrast in many browsers based on the background its placed on. This is not as much of an issue for navigation but it is for smaller components its helpful since we don't know what the background might be. I wrote up a detailed blog post here about the differences/trade-offs https://coryrylan.com/blog/dont-override-css-outline-focus-styles Overall if the native outline can work I'd recommend adding it with these changes as well.

Reply:
I addressed this per our discussion by moving the style for focus back to outline with the pseudo element and an inset to prevent outline clipping.